docs/providers/provider-protocol-reference.md

```diff
@@ -0,0 +1,243 @@
+---
+title: Provider Protocol Reference
+category: 63e0c7cf7166bf006d3b4117
+slug: provider-protocol-reference
+---
+
+# Provider Protocol Reference (v2)
+
+All functions described here are bindings generated with fp-bindgen.
+The key/values to use for these payloads are exactly the ones you
```

Comment:
This is quite the nitpick, but I notice on the website that newlines are inserted as hard `<br>` tags. This is rather jarring, especially given that different paragraphs seem to have different line lengths in this file. I think we should configure the converter to not insert those tags, but if that's not an easy fix for some reason, at least the line lengths in this file should be consistent. (@keturiosakys)